211 Blog for teal and Posit Shiny Contest 2024 (#212)
pharmaverse/blog: posts/2024-08-19_teal_submission_.../teal_submission_for__posit__shiny__contest_2024.qmd

---
title: "teal and Posit Shiny Contest 2024"
author:
  - name: Dony Unardi
  - name: Pawel Rucki

> description: "this blog is to announce teal category in Post Shiny Contest 2024 and to inform users on resources to make their teal app for the submission."

Reviewer: ```suggestion
description: "Introducing the {teal} category in the Post Shiny Contest 2024 and informing users on resources to make their teal app for the submission."
```

> date: "2024-08-19"

Reviewer: Please update the date to the final publication date

# please do not use any non-default categories.
# You can find the default categories in the repository README.md

> categories: [Shiny, Technical]

Reviewer: I think the Technical tag is not appropriate, please replace with "Community"
